src/service/programService.js

```diff
@@ -406,8 +405,64 @@ function unlistPublishProgram(req, response) {
         return response.status(400).send(errorResponse(errObj,errCode+errorCodes.CODE4));
       }
     };
-
-    programServiceHelper.copyCollections(res, data.request.channel, req.headers, cb);
+      model.configuration.findOne({
```

Comment:
@bharatkashyap Can you please create a function to get a configuration that will take input as key and return value as output. So we can use that function in some other place as well.

Reply:
@pallakartheekreddy - Have done